camille/util/baze_iterator.py

```diff
@@ -119,3 +126,36 @@ def __iter__(self):
 
     def __len__(self):
         return len(self.it)
+
+
+def _get_files(src_dirs, tags):
```

Comment:
There are now two `_get_files` functions, are the different?

Reply:
Yes, this one returns all files while the other ones does filtering